include/glow/Base/Tensor.h

```diff
@@ -1045,6 +1045,53 @@ inline size_t getFlattenedOffset(llvm::ArrayRef<dim_t> strides,
   return index;
 }
 
+/// Helper function which \returns true if a slice with the shape \p sliceShape
+/// references fully a tensor with the shape \p tensorShape. This happens when
+/// the shapes are the same.
+inline bool isSliceFull(llvm::ArrayRef<dim_t> sliceShape,
```

Comment:
Isn't this the same as just checking if `sliceShape == tensorShape`? I don't think this is necessary to add, can just compare the two inputs directly there.

Reply:
Yes you`re right. I think I over-engineered this. I removed this and replaced with equality checks in the unit tests.